src/pages/home/HomeScreen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { View, Text } from 'react-native'
+
+import styles from './style'
+import { activeTabName } from '../../redux/action/activeTabName'
+
+class HomeScreen extends React.Component {
+    render() {
+        return (
+            <View style={styles.background}>
+                <Text>Home</Text>
+            </View>
+        )
+    }
+}
+
+const mapStateToProps = (state) => {
```

Comment:
Currently, you can remove the redux connection.
If we need to use Redux in the future, we can add then.